### DISCUSSION TITLE
Consider publishing fossify apps on the accressent store

### DISCUSSION THREAD

**Discussion by realeatham** [FossifyOrg/General-Discussion#228] [General]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/General-Discussion/issues) or [closed](https://github.com/FossifyOrg/General-Discussion/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/General-Discussion/discussions) or [closed](https://github.com/FossifyOrg/General-Discussion/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs), in the [README](https://github.com/FossifyOrg/General-Discussion/blob/master/README.md) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report may get dismissed otherwise.**
- [X] This issue contains only one feature request.
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Which apps should have this feature?

All Fossify Apps

### Feature description

Add fossify apps to the Accrescent store. See info about them [here](https://accrescent.app/).  This store is recommended by grapheneos. It would be good to add fossify to more stores so it is easier to get for note people who might not want gplay and f-droid. It would also be a good idea to publish to more oem stores like the Amazon store and appgallary eventually, as those devices do not have gplay installed.

### Why do you want this feature?

See above

**Comment by naveensingh** [comment]
There are no plans for this as of now. Maybe in the future.

More stores = more work.

**Comment by ghost** [comment]
I would love to see that too!

[Accresent features](https://accrescent.app/features) I think matter for Fossify Apps.
 
Security: 
[Signed repository metadata](https://accrescent.app/features#signed-metadata) 
[App signing key pinning](https://accrescent.app/features#key-pinning) 
[Name verification](https://accrescent.app/features#name-icon-verification)
 
Privacy: 
[No remote APK signing](https://accrescent.app/features#no-remote-signing) 
[No accounts](https://accrescent.app/features#no-accounts)
